guides/source/configuring.md

```diff
@@ -760,6 +760,21 @@ Specifies if an error should be raised if the order of a query is ignored during
 
 Controls whether migrations are numbered with serial integers or with timestamps. The default is `true`, to use timestamps, which are preferred if there are multiple developers working on the same application.
 
+#### `config.active_record.migration_strategy`
+
+Controls the strategy class used to perform schema statement methods in a migration. The default class
+delegates to the connection adapter. Custom strategies should inherit from `ActiveRecord::Migration::ExecutionStrategy`:
+
+```ruby
+class CustomMigrationStrategy < ActiveRecord::Migration::ExecutionStrategy
```

Comment:
Is this fully working example? `ExecutionStrategy` doesn't delegate method to `connection` so it seems this example strategy supports only `drop_table` and other calls (like `create_table`) will result into `NoMethodError`. :thinking: Would make sense to inherit from `DefaultStrategy` in here?

Reply:
This was just a rough example -- I definitely think it makes sense for it to inherit from `DefaultStrategy`, thanks!